.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,26 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

Comment:
https://github.com/marketplace/actions/pypi-publish use the pypi publish action with a scoped token for the cfn-python-lint package. simplifies to where you just need to include the command like here: https://github.com/aws-cloudformation/cloudformation-cli/blob/master/.github/workflows/pypi-release.yaml#L26